src/validators/ResponseValidator.js

```diff
@@ -17,7 +17,8 @@ function responseValidator(response, data) {
         if (bodySchema.validate) {
             validator = bodySchema;
         } else {
-            const responseContentType = data.headers['Content-Type'] || data.headers['content-type'] || DEFAULT_CONTENT_TYPE;
+            const responseContentTypeHeader = data.headers['Content-Type'] || data.headers['content-type'] || DEFAULT_CONTENT_TYPE;
+            const responseContentType = responseContentTypeHeader.split(';')[0]; // This is to filter out things like charset
```

Comment:
maybe it's better to also trim the result of the result of the split?

Reply:
done